src/bot/handlers/registration.py

```diff
@@ -1,32 +1,25 @@
-from telegram import InlineKeyboardButton, InlineKeyboardMarkup, Update
+from telegram import Update
 from telegram.constants import ParseMode
-from telegram.ext import Application, CommandHandler, ContextTypes
+from telegram.ext import Application, CallbackQueryHandler, CommandHandler, ContextTypes
 
-from src.bot.constants import callback_data, commands
+from src.bot.constants import commands
+from src.bot.keyboards import get_confirm_keyboard, get_start_keyboard
 from src.core.logging.utils import logger_decor
 from src.core.services.user import UserService
 
 
 @logger_decor
 async def start_command(update: Update, context: ContextTypes.DEFAULT_TYPE):
+    telegram_id = update.effective_chat.id
```

Comment:
Зачем эта переменная? Можно обойтись без нее.

Reply:
Лучше использовать user id
```suggestion
    telegram_id = update.effective_user.id
```